go.mod

```diff
@@ -80,7 +80,7 @@ require (
 	github.com/hashicorp/vault-plugin-auth-jwt v0.6.2
 	github.com/hashicorp/vault-plugin-auth-kerberos v0.1.5
 	github.com/hashicorp/vault-plugin-auth-kubernetes v0.6.1
-	github.com/hashicorp/vault-plugin-auth-oci v0.5.4
+	github.com/hashicorp/vault-plugin-auth-oci v0.5.3-beta1.0.20200611202039-7f7ad0f784bc
```

Comment:
As we discussed, might be good to push a new tag to the OCI master branch so this doesn't look quite so odd?

Reply:
Ah, yes. The `beta1` showed up. I missed this. Will push a tag and update this PR.

Reply:
I've tagged vault-plugin-auth-oci [v0.5.5](https://github.com/hashicorp/vault-plugin-auth-oci/releases/tag/v0.5.5) brough that into `go.mod` now.

Reply:
Thanks!